packages/webapp/pages/account/profile.tsx

```diff
@@ -39,6 +39,7 @@ const AccountProfilePage = (): ReactElement => {
   const onSuccess = () => displayToast('Profile updated');
   const { updateUserProfile, isLoading, hint } = useProfileForm({ onSuccess });
   const { user, updateUser } = useContext(AuthContext);
+  const [coverImage, setCoverImage] = useState<string>(null);
```

Comment:
We could, but then we also need `useEffect` to check if `user.cover` did not change.

Reply:
Hmm would work if you just set it as default instead of null here no?
Don't think you need to use effect for that?

Reply:
The state does not update if props update from what I know, unless react has changed that recently 👀 . So if we set the null to user.cover it would help to set that as initial value, but we would still need useEffect to keep the state updated if user.cover changes. 
But this would give us the possibility to not check the user.cover anymore down below in the code, but we would only depend on coverImage.

Reply:
Yeah, default state is set only once at component mount. 

Do we need to sync anything with useEffect?, from logic it looks like `const currentCover = coverImage || user.cover` should be enough, and then you check `currentCover` in the render code below?

Reply:
Yup, with the current implementation, the `useEffect` is not needed and we can avoid using it, which is my preferred solution.

Reply:
Current one is fine indeed :)